templates/custom_search.pages.inc

```diff
@@ -29,12 +29,12 @@ function custom_search_preprocess_search_results(&$variables) {
         $keys = backdrop_substr($keys, 0, strpos($keys, 'type:') - 1);
       }
       // Get Custom Search authorized types.
-      $searchable_node_types = array();
+      $searchable_node_types = $defined_types = array();
```

Comment:
Since you are not using `count()` anymore, this initial assignment of `$defined_types` is not needed here. It will be initialized by the following line in the `if` statement - either to NULL or to an array, depending on whether there is a value stored in the json file. 

Alternatively, you can keep `count()` and instead change the next line to 
```php
if ($defined_types = (array) $config->get('current_node_types')) {
```

In either case, the initialization of `$defined_types` in the line above is not needed.

Reply:
That's true. Even more minimal change